src/tests/integration/algorithms/test_onedot.py

```diff
@@ -15,28 +15,28 @@
 class TestOneDot(unittest.TestCase):
 
     @staticmethod
-    def one_dot(fig = None):
+    def one_dot(fig=100, verbose=1):
         nr_dots = 3
-        station = initialize(reinit=True, nr_dots=nr_dots, maxelectrons=2, verbose=0)
+        station = initialize(reinit=True, nr_dots=nr_dots, maxelectrons=2, verbose=verbose)
         gates = station.gates
 
         gv = {'B0': -300.000, 'B1': 0.487, 'B2': -0.126, 'B3': 0.000, 'D0': 0.111, 'O1': -0.478, 'O2': 0.283, 'O3': 0.404, 'O4': 0.070,
               'O5': 0.392, 'P1': 0.436, 'P2': 0.182, 'P3': 39.570, 'SD1a': -0.160, 'SD1b': -0.022, 'SD1c': 0.425, 'bias_1': -0.312, 'bias_2': 0.063}
-        gates.resetgates(gv, gv, verbose=0)
+        gates.resetgates(gv, gv, verbose=verbose)
 
         start = -250
         scanjob = scanjob_t({'sweepdata': dict(
             {'param': 'B0', 'start': start, 'end': start + 200, 'step': 4., 'wait_time': 0.}), 'minstrument': ['keithley3.amplitude']})
         scanjob['stepdata'] = dict({'param': 'B1', 'start': start, 'end': start + 200, 'step': 5.})
         data = scan2D(station, scanjob)
 
-        x = onedotGetBalance(dataset=data, verbose=1, fig=fig)
+        x = onedotGetBalance(dataset=data, verbose=verbose, fig=fig)
         results = x[0]
-        _, _ = onedotGetBalanceFine(impixel=None, dd=data, fig=fig)
-        plot_onedot(results, ds=data, fig=None, verbose=1)
+        _, _ = onedotGetBalanceFine(impixel=None, dd=data, fig=fig+100)
+        plot_onedot(results, ds=data, fig=fig+200, verbose=verbose)
```

Comment:
A figure with the same number will overwrite/replace an earlier figure with number 100 within the same test. I thought to see them all in the end (at the show statement) we should use unique numbering.

I changed it back to what it was originally, maybe there is a reason to overwrite/replace figures I don't know of.

Reply:
Ah, got it. I see your point. When running the integration tests you will just see some windows popping up, for which it is not really needed to use new figures. But if you just want to run this single integration test 'by hand' it would indeed be nice, to see all the output.